app/src/layouts/appLayout/appSidebar/appSidebar.jsx

```diff
@@ -100,7 +119,8 @@ export class AppSidebar extends Component {
         message: <FormattedMessage id={'Sidebar.dashboardsBtn'} defaultMessage={'Dashboards'} />,
       },
       {
-        onClick: () => this.onClickButton(SIDEBAR_EVENTS.CLICK_LAUNCHES_BTN),
+        onClick: () =>
+          this.onClickButton(SIDEBAR_EVENTS.clickLaunchesBtn(this.getFiltersStatistic())),
```

Comment:
I believe that tracking the number of user filters needs to be done elsewhere.
The launches page is responsible for displaying the launches and their filters, and it might be better to keep track of the number of filters when the launches page is mounted.